features/in_onderzoek.feature

```diff
@@ -532,15 +635,22 @@ Functionaliteit: in onderzoek
       | burgerservicenummer | 555550001                          |
       | fields              | burgerservicenummer,verblijfplaats |
       Dan heeft het kind met burgerservicenummer '555550002' de volgende 'verblijfplaats.inOnderzoek' gegevens
-      | naam                        | waarde     |
-      | adresregel1                 | true       |
-      | adresregel2                 | true       |
-      | adresregel3                 | true       |
-      | datumInschrijvingInGemeente | true       |
-      | gemeenteVanInschrijving     | true       |
-      | land                        | true       |
-      | datumIngangGeldigheid       | true       |
-      | datumIngangOnderzoek        | 2022-03-07 |
+      | naam                             | waarde       |
+      | adresregel1                      | true         |
+      | adresregel2                      | true         |
+      | adresregel3                      | true         |
+      | land                             | true         |
+      | datumIngangGeldigheid            | true         |
+      | datumIngangOnderzoek.datum       | 2022-03-07   |
+      | datumIngangOnderzoek.type        | Datum        |
+      | datumIngangOnderzoek.langFormaat | 7 maart 2022 |
+      En heeft de persoon met burgerservicenummer '5555550001' de volgende 'inOnderzoek' gegevens
+      | naam                                     | waarde       |
+      | datumInschrijvingInGemeente              | true         |
```

Comment:
dit is niet terecht, er is met fields toch niet gevraagd om datumInschrijvingInGemeente en gemeenteVanInschrijving en die zijn toch ook niet geleverd?